### PR TITLE
Updating LUCA database version in Omamer data manager

### PR DESCRIPTION
**Pull request by SaimMomin12**
FOR CONTRIBUTOR:
* [x] I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] License permits unrestricted use (educational + commercial)
* [ ] This PR adds a new tool or tool collection
* [x] This PR updates an existing tool or tool collection
* [ ] This PR does something else (explain below)

Based on the LUCA database update request by @tbrown91 for OMArk, this PR concerns with updating the data manager to install the latest version of the OMark databases while retaining the older databases version.

### CODE REVIEW

File: data_managers/data_manager_omamer/data_manager/omamer.xml [hunk 1]
```diff
@@ -15,7 +15,8 @@
             <option value="Primates">Primates</option>
             <option value="Viridiplantae">Viridiplantae</option>
             <option value="Metazoa">Metazoa</option>
-            <option value="LUCA">LUCA</option>
+            <option value="LUCA_v0.2.5">LUCA_v0.2.5</option>
```
Comment (bgruening) [inline data_managers/data_manager_omamer/data_manager/omamer.xml:18]: please add the others as well ...

**Comment by tbrown91** [comment]
image

**Comment by tbrown91** [comment]
Please only include the v2 databases. The others will not work with this version of OMArk/OMAmer

**Comment by SaimMomin12** [comment]
We currently have all v2 databases added to the list

**Comment by tbrown91** [comment]
On the OMArk github page, they recommend using the LUCA.h5 file, so maybe it would be useful to have the *.h5 and *v2.0.0.h5 files, but not the v0

**Comment by SaimMomin12** [comment]
Okay, I will add them all shortly

**Comment by bgruening** [comment]
Is the .h5 file not the latest version? So identical with v2.0.0.h5 ...?

**Comment by tbrown91** [comment]
Not clear, and there are more .h5 than .v2.0.0.h5 files

**Comment by SaimMomin12** [comment]
https://github.com/DessimozLab/OMArk?tab=readme-ov-file#omamer-database

Does this mean .h5 ones are the complete databases? I'm confused now

**Comment by tbrown91** [comment]
No LUCA (Last Universal Common Ancestor) is the most complete database and includes metazoa, viridiplantae, etc

**Comment by SaimMomin12** [comment]
Okay, but we cannot find the difference between the .h5 ones and the other version ones?

**Comment by tbrown91** [comment]
Yes, I'm struggling to find an explanation

**Comment by bgruening** [comment]
I assume there is a detailed version (v2.0.0.h5) and a link to a general-latest version. With this general link, you always get the latest version, but you don't know which version that is.

So I would recommend always taking the link with a clear version in the name.

**Comment by SaimMomin12** [comment]
@bgruening So based on your last comment, we eliminate the `Primates.h5, Viridiplantae.h5, Metazoa.h5, Saccharomyceta.h5 and Homininae.h5` and only keep the ones with the clear version in their name?

**Comment by bgruening** [comment]
That would be my recommendation. With this we at least know what we get ... a versioned DB.

**Pull request merged**